.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
@derhuerst you mean...

```json
	"main":        "./src/index.js",
	"files":       ["./src/index.js"],
```

...?

Good point! I forgot that very important change 😮. I believe the module won't be able to be imported after installed if I don't fix that. Right?